drivers/network/afd/afd/write.c

```diff
@@ -398,7 +398,7 @@ AfdConnectedSocketWriteData(PDEVICE_OBJECT DeviceObject, PIRP Irp,
                 }
             }
 
-            ExFreePool( TargetAddress );
+            ExFreePoolWithTag(TargetAddress, TAG_AFD_TRANSPORT_ADDRESS);
```

Comment:
TAG_AFD_TDI_CONNECTION_INFORMATION

Reply:
And I thought I was careful.. Thanks! :)